src/Utilities/StdConstexprHelpers.hpp

```diff
@@ -0,0 +1,116 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/*!
+ * \ingroup Utilities
```

Comment:
What is this comment being applied to?

Reply:
Nothing, and it's deleted in the next commit